docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx

```diff
@@ -0,0 +1,281 @@
+
+Optimizing videos is crucial in web development for maintaining site performance and enhancing user experience. This document dives into video optimization principles for Next.js web applications, highlighting effective storage and streaming methods to improve video delivery.
+
+## Using `<video>` and `<iframe>`
+
+Effectively embedding video content is crucial for a dynamic web experience. This involves using the HTML **`<video>`** tag for direct video files and **`<iframe>`** for external platform-hosted videos. Understanding their use in the context of Next.js's rendering capabilities ensures optimized performance and an enhanced user experience.
+
+### `<video>`
+
+In Next.js, the HTML [`<video>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video) tag is essential for embedding video content directly into web pages. It's especially effective for self-hosted or directly served video files, allowing extensive control over playback and appearance. Integrating the `<video>` tag into your JSX code in Next.js enables a native video player experience on your web pages.
+
+Here's an example:
+
+```html
+<video width="320" height="240" controls preload="none">
+  <source src="/path/to/video.mp4" type="video/mp4" />
+  <track
+    src="/path/to/captions.vtt"
+    kind="subtitles"
+    srclang="en"
+    label="English"
+  />
+  Your browser does not support the video tag.
+</video>
+```
+
+### Common `<video>` Tag Attributes
+
+| Attribute  | Description                                                                                               | Example Value                              |
+| ---------- | --------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| `src`      | Specifies the source of the video file.                                                                   | `<video src="/path/to/video.mp4"></video>` |
+| `width`    | Sets the width of the video player.                                                                       | `<video width="320"></video>`              |
+| `height`   | Sets the height of the video player.                                                                      | `<video height="240"></video>`             |
+| `controls` | If present, it displays the default set of playback controls.                                             | `<video controls></video>`                 |
+| `autoplay` | Automatically starts playing the video when the page loads. Note: Autoplay policies vary across browsers. | `<video autoplay></video>`                 |
+| `loop`     | Loops the video playback.                                                                                 | `<video loop></video>`                     |
+| `muted`    | Mutes the audio by default. Often used with `autoplay`.                                                   | `<video muted></video>`                    |
```

Comment:
> However the video above still won’t autoplay on iOS Safari, Safari requires us to add the playsinline attribute.

Found this from a component we're using:

```
<video
  autoPlay
  loop
  muted
  playsInline
/>
```

We might want to include that guidance here? Or include this set of default props on the self-hosted video example.